reporting.md

```diff
@@ -0,0 +1,615 @@
+%%%
+
+   Title = "SMTP TLS Reporting"
+   abbrev = "SMTP-TLSRPT"
+   category = "std"
+   docName = "draft-ietf-uta-smtp-tlsrpt-00"
+   ipr = "trust200902"
+   area = "Applications"
+   workgroup = "Using TLS in Applications"
+   keyword = [""]
+
+   date = 2016-04-18T00:00:00Z
+
+   [[author]]
+   initials="DM"
+   surname="Margolis"
+   fullname="Daniel Margolis"
+   organization="Google, Inc"
+     [author.address]
+     email="dmargolis (at) google.com"
+   [[author]]
+   initials="AB"
+   surname="Brotman"
+   fullname="Alexander Brotman"
+   organization="Comcast, Inc"
+     [author.address]
+     email="alexander_brotman (at) cable.comcast (dot com)"
+   [[author]]
+   initials="BR"
+   surname="Ramakrishnan"
+   fullname="Binu Ramakrishnan"
+   organization="Yahoo!, Inc"
+     [author.address]
+     email="rbinu (at) yahoo-inc (dot com)"
+   [[author]]
+   initials="JJ"
+   surname="Jones"
+   fullname="Janet Jones"
+   organization="Microsoft, Inc"
+     [author.address]
+     email="janet.jones (at) microsoft (dot com)"
+   [[author]]
+   initials="MR"
+   surname="Risher"
+   fullname="Mark Risher"
+   organization="Google, Inc"
+     [author.address]
+     email="risher (at) google (dot com)"
+
+%%%
+
+.# Abstract
+
+SMTP Mail Transfer Agents often conduct encrypted communication on the Internet
+through the use of Transport Layer Security (TLS). Because the STARTTLS protocol
+is opportunistic in nature, malicious and misconfigured intermediaries can
+interfere with the successful establishment of suitable encryption, and such
+interference is not always detectable by the receiving server. This document
+provides transparency into failures in the SMTP MTA Strict Transport Security
+policy (TODO: Add ref), negotiation of STARTTLS [@!RFC3207], and the DNS-Based
+Authentication of Named Entities (DANE, [@!RFC6698]).
```

Comment:
Wow, that's wordy. Can we just say, 

"...provides transparency into SMTP STS [ref], STARTTLS [ref], and DANE [ref] failures."

instead?